examples/with-react-jss/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
```

Comment:
This seems unneeded 🤔

Reply:
I get this error when I don't have this file: 
```sh
Module build failed: Error: [BABEL] /Users/henri/Coding/next.js/examples/with-react-jss/pages/_app.js: You gave us a visitor for the node type PrivateName but it's not a valid type
```

Reply:
You possibly have a `.babelrc` in an upward directory 🤔

Reply:
Yeah, that one: https://github.com/zeit/next.js/blob/canary/examples/.babelrc

Reply:
Ahhh yeah that's fine. Users download only the example itself. Either way it'll work 👍